policy/lib/tekton/task_test.rego

```diff
@@ -0,0 +1,134 @@
+package lib.tkn
+
+import future.keywords.if
+import future.keywords.in
+
+import data.lib
+import data.lib.bundles
+
+test_latest_required_tasks if {
+	expected := {t | some t in _expected_latest.tasks}
+	lib.assert_equal(expected, latest_required_tasks) with data["required-tasks"] as _time_based_required_tasks
```

Comment:
Do we need something like:
```suggestion
	lib.assert_equal(expected, latest_required_tasks) with data["required-tasks"] as _time_based_required_tasks with data["config"] as {"policy": {"when_ns": "2022-11-01T00:00:00Z"}}
```
To prevent time travel?

Reply:
We're good until 2099 at which point we'll be retired and likely dead. This will be my legacy for the future generations.